0xOpenBytes/ios-base
# Leif/task/fix swift lint

## Description

Updates the .swiftlint.yml file to not lint SPM dependencies. Regenerate project to include UITests. Remove the Xcode project and add it to the .gitingore

## Testing Steps

- Open the App
- Build the App
- Notice only Swiftlint warnings are related to TODOs 

## Areas of Code Impacted

- .swiftlint.yml
- scripts
- Removed Xcode project
- .gitignore
